src/dotnet/APIView/ClientSPA/src/app/_components/code-panel/code-panel.component.ts

```diff
@@ -619,6 +634,43 @@ export class CodePanelComponent implements OnChanges{
     }
   }
 
+  copyReviewTextToClipBoard() {
```

Comment:
are there any tests we can add for this?

Reply:
Added